src/main/java/org/datadog/jenkins/plugins/datadog/DatadogGlobalConfiguration.java

```diff
@@ -274,6 +261,10 @@ private boolean validateTargetHost(String targetHost) {
         return StringUtils.isNotBlank(targetHost);
     }
 
+    public static boolean validateURL(String targetURL) {
+        return targetURL.contains("http");
```

Comment:
```suggestion
        return StringUtils.isNotBlank(targetURL) &&  targetURL.contains("http");
```
If we make this change the validation will be similar to `validatePort` and the extra `isNotBlank` checks can be removed from the calling functions.

Reply:
Updated!